src/components/NavBar/SuggestionRow.tsx

```diff
@@ -100,6 +102,15 @@ export const CollectionRow = ({
   )
 }
 
+function useBridgedAddress(token: FungibleToken): [string | undefined, number | undefined, string | undefined] {
```

Comment:
I think we should move this helper fn to the same util files as `getTokenDetailsURL`

Reply:
I'm gonna do some refactoring of where that util lives later. Right now, that util is in the graphql section of the code, it'd probably be weird to have this FungibleToken data structure used there too since it's from a different endpoint